exchanges/gateio/ws_ob_update_manager.go

```diff
@@ -0,0 +1,152 @@
+package gateio
```

Comment:
Just for curiosity's sake, what made our existing orderbook buffer insufficient?

Reply:
It doesn't allow for fetching an initial state and handing of what's required, I guess with time and effort it could be upgraded. 🤷

Reply:
Yeah had this PR open for like 8 months or so trying to abstract and consolidate this very thing across all the exchanges, but it never got reviewed so I [closed it.](https://github.com/thrasher-corp/gocryptotrader/pull/1481)

Reply:
I thought I had but turns out it was `Pending` 😄 well even less reason for such a consolidation then